crates/proto/src/xfer/dnssec_dns_handle.rs

```diff
@@ -542,7 +542,7 @@ where
                 // must be covered by at least one DS record
                 .any(|(ds_name, ds_rdata)| {
                     if ds_rdata.covers(&rrset.name, key_rdata).unwrap_or(false) {
-                        if log::log_enabled!(log::Level::Debug) {
+                        if tracing::level_enabled!(tracing::Level::DEBUG) {
```

Comment:
similarly, this should use
```suggestion
                        if tracing::event_enabled!(tracing::Level::DEBUG) {
```

Reply:
although, actually, in this case, i don't understand why the `level_enabled!` macro is necessary here --- the `debug!` macro will already not be enabled if `DEBUG` is disabled. the macro should only be necessary to guard some extra computation that's outside of the actual event macro.

Reply:
I switched to the simpler, `tracing::enabled` which looks generally simpler :)

Reply:
Oh, and that one wasn't needed...